libs/ledger-live-common/src/families/filecoin/bridge/utils/addresses.ts

```diff
@@ -1,150 +1,126 @@
+import blake from "blakejs";
 import base32Decode from "base32-decode";
-import { blake2bInit, blake2bUpdate, blake2bFinal, Blake2bCTX } from "blakejs";
-import leb128 from "leb128";
-import { log } from "@ledgerhq/logs";
 
-type PrefixObjType = Readonly<{ f: "mainnet" } & { t: "testnet" }>;
-type PrefixType = keyof PrefixObjType;
+// @ts-ignore
+import BN from "bn.js";
 
-type ProtocolsObjType = Readonly<
-  { "0": "ID" } & { "1": "SECP256K1" } & { "2": "Actor" } & {
-    "3": "BLS";
-  }
->;
+// @ts-ignore
+import leb from "leb128";
```

Comment:
Why do we also need a ts-ignore here ?

Reply:
The ts-ignore is because of the lack of definition file :/